### DISCUSSION TITLE
Does Validation-and-Serialization document's outdated?

### DISCUSSION THREAD

**Discussion by hasunpark** [fastify/fastify#5457] [Q&A] [answered]
there's an paragraph 
> Validation will only be attempted if the content type is application-json, as described in the documentation for the [content type parser](https://fastify.dev/docs/latest/Reference/ContentTypeParser/).

but when i move to [content type parser](https://fastify.dev/docs/latest/Reference/ContentTypeParser/). and tried to find about validation or serialization. Can't find about it

**Comment by mcollina** [comment, marked as answer]
I don't really understand what you are looking for. But definitely, this is a bug. I've opened https://github.com/fastify/fastify/issues/5459 to track it.
